src/Integration.Vsix/Analysis/AnalysisStrings.resx

```diff
@@ -121,7 +121,7 @@
     <value>Suppressions have been updated. Open documents will be re-analysed.</value>
   </data>
   <data name="ConfigMonitor_UserSettingsIgnoredForConnectedModeLanguages" xml:space="preserve">
-    <value>Note: user settings are ignored for languages that support Connected Mode i.e. C and C++</value>
+    <value>User settings change is ignored: current workspace is in Connected Mode.</value>
```

Comment:
This will be shown only when we're in connected mode.
The message below will be shown only when we're in standalone mode.